File: tbshop-interface/src/main/java/com/shop/service/ShopService.java
package com.shop.service;

import java.util.List;

import com.shop.pojo.PageResult;
import com.shop.pojo.TbDing;
import com.shop.pojo.TbShop;

public interface ShopService {

	PageResult shopList(Integer page, Integer rows);

	void changeZt(TbShop shop);

	void add(TbShop shop);

	List<TbDing> dingList();

	TbShop shopHui(Integer sid);

	void shopAdd(TbDing ding);

}

File: tbshop-web/src/main/webapp/js/service/shopService.js
/**
 * 
 */
app.service('shopService',function($http){
	this.shopList=function(page,rows){
		return $http.get('../month/shopList.do?page='+page+"&rows="+rows);
	}
	this.changeZt=function(entity){
		return $http.post('../month/changeZt.do',entity);
	}
	this.add=function(entity){
		return $http.post('../month/add.do',entity);
	}
	this.findUser=function(){
		return $http.get('../month/findUser.do');
	}
	this.dingList=function(){
		return $http.get('../month/dingList.do');
	}
	this.shopHui=function(sid){
		return $http.get('../month/shopHui.do?sid='+sid);
	}
	this.shopAdd=function(entity){
		return $http.post('../month/shopAdd.do',entity);
	}
})